multidim-interop/go/v0.26/Makefile

```diff
@@ -0,0 +1,20 @@
+image_name := go-v0.26
+commitSha := 59a14cf3194d5d057c45cb1dbc7b1af3a116bc7a
```

Comment:
Any way we can use the tag name here?

Reply:
I believe tags are mutable. The goal is for this always to work. If we could use a combination tag and sha I would be okay with that. Wdyt?

Reply:
On the Git level they are, but when using Go, they're not. The Go proxy makes sure that you can effectively never update a tag.

Reply:
So it should be fine to just use tags here, as it makes it more readable. No strong feelings though, we can update this in a later PR.

Reply:
> On the Git level they are, but when using Go, they're not. The Go proxy makes sure that you can effectively never update a tag. 

This hash is used to download a Git repo though. How is the Go proxy involved here? What am I missing?

Reply:
Yeah, but that doesn't guarantee that using a tag is immutable here. What I can do is use a tag and then compare the sha256 output of the artifact with some expected value. That way we get the tag semantics and the pinning semantics.